integration_test/tests/power_bi_tests.rs

```diff
@@ -144,7 +144,7 @@ mod integration {
             in_connection_string_encoded.push(0);
 
             let str_len_ptr = &mut 0;
```

Comment:
Documentation suggests the buffer size should be 1024 minimum, shouldn't have actually be a problem, however.